### PR TITLE
Milestone 1: Clone of NY Times website using CSS Float as the positioning technique.

### PR DESCRIPTION
**Pull request by tiagoit**
# NY Times Post using CSS Float

Copy of an NY Times postusing CSS Float positioning. (an Microverse Project)

### Technology

HTML, CSS

### Authors

- **Tiffany Arellano** - [@yirano](https://github.com/yirano)
- **Tiago Ferreira** - [@ferreirati](https://github.com/ferreirati)

### Links

- **Assignment page** - [The Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements)
- **Live** - [Github pages](https://ferreirati.github.io/mv-03-htmlcss-nytimes/src)

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Nice work, team ferreirati & yirano, great improvement :clap: 

### However, there are few adjustments to make. 
- [x]  The **```editor's pick section```** is overlapping the articles, kindly fix that.

- [x] The bottom part of your footer,  should have a fixed position (it sticks to the bottom part of the screen); it is for implementing the **additional ```logging in & creating free account```**. 
 [**use this snap shot  :point_down: as reference to effect the changes listed above: 1, 2**]

Screenshot from 2019-09-19 06-21-56

This is the snap shot from [newTimes](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) **```use it as reference for implementing the change on the footer section and the aside section of your project```**

Screenshot from 2019-09-19 06-07-28

### After implementing these changes submit another form for code review.

### This project is not approved :stop_sign:

**Review by Forison — CHANGES_REQUESTED** [review]
# Nice try, @Tiffany Arellano &&  @Tiago Ferreira 🙂
Your project looks good but with these few tweaks, it will 👌.
make and the following changes and submit them for review. 🙂
## **[REQUIRED]** FOOTER🙂
The footer must be fixed to the bottom. check the original [page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) 
>It has a fixed row at the bottom with a caret up this inscription '' Access more of the times by creating an account"
## **[REQUIRED]** most popular section && More in Space & Cosmos🙂
The two sections are too wide apart causing your page to have a horizontal scroll. please fix it.🙂
It looks very messy at the moment check your margins and paddings.
## **[OPTIONAL]** NAVBAR🙂
Your navbar has a transparent background-color, change it to white(#fff)
> WILL MAKE YOUR PROJECT LOOK BETTER
##### Guys, please fix the errors I have pointed out and submit your project for another review
feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
Happy coding🙂🙂

**Comment by tiagoit** [comment]
> # Nice try, @tiffany Arellano && @tiago Ferreira 
> Your project looks good but with these few tweaks, it will .
> make and the following changes and submit them for review. 
> 
> ## **[REQUIRED]** FOOTER
> The footer must be fixed to the bottom. check the original [page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)
> 
> > It has a fixed row at the bottom with a caret up this inscription '' Access more of the times by creating an account"
> 
> ## **[REQUIRED]** most popular section && More in Space & Cosmos
> The two sections are too wide apart causing your page to have a horizontal scroll. please fix it.
> It looks very messy at the moment check your margins and paddings.
> 
> ## **[OPTIONAL]** NAVBAR
> Your navbar has a transparent background-color, change it to white(#fff)
> 
> > WILL MAKE YOUR PROJECT LOOK BETTER
> 
> ##### Guys, please fix the errors I have pointed out and submit your project for another review
> feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
> Happy coding

Hello @Forison, thank you for the CR, the suggestions have been implemented.

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+        </span>
+        <span class="title">
+            The Sly Superpower of ‘Hustlers’
+        </span>
+      </div>
+    </aside>
+
+    <section class="bottom-sec">
+      <div class="more-in-category">
+        <div class="section-title">More in Space & Cosmos</div>
```
Comment (bolah2009) [inline src/index.html]: Why is this not a heading element? :thinking: 

Consider using `h2-h6` element here instead. `div` has no semantic meaning and this part of the section is obviosly the **heading** of the section.

> Validate your HTML markup with https://validator.w3.org/#validate_by_upload.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+          <li>Times Journeys</li>
+          <li>Manage My Account</li>
+        </ul>
+
+        <ul>
+          <li>SUBSCRIBE</li>
+          <li>Home Delivery</li>
+          <li>Digital Subscriptions</li>
+          <li>Crossword</li>
+          <li>Cooking</li>
```
Comment (bolah2009) [inline src/index.html]: There are some missing icons here, see below:

| Current | Preffered |
|:----:|:-----:|
| nyt-current  | nyt footer preffered  |

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+  width: 110%;
+  left: 0;
+  top: 130px;
+}
+
+.diagram-img-caption {
+  display: inline-block;
+  width: 204px;
+  height: 100px;
+  margin: 35px 10px;
```
Comment (bolah2009) [inline src/style.css]: Let's adjust this margin for positioning the text appropriately. See Below:

nyt-inflation

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+section.bottom-sec > .more-in-category {
+  width: 66%;
+  float: left;
+  border-top: 1px solid black;
+  margin: 40px 0;
+}
+
+section.bottom-sec > .more-in-category .article-thumb {
+  display: inline-block;
+  width: 33%;
```
Comment (bolah2009) [inline src/style.css]: Let's reduce this to `30%`.

Currently, the **More in Space & Cosmos** section is messed up with a zoom percentage increase. I initially viewed it with `100%` zoom and notice this. Changing to `30%` will reduce the unnecessary space this element is taking and consequently allowing three elements to be in a single row. See below:

nyt-misnc

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome job Team :clap: 


### The [clone](https://raw.githack.com/ferreirati/mv-03-htmlcss-nytimes/aa3decad32aa339d7c7859c93bdb365a56832833/src/index.html) looks good to me, uses CSS Positioning effectively. :clap: 

There are however a few required changes  to make the project even better  :blush:  :
 - Kindly rename the PR (to the name of the milestone) and the PR description to a brief summary of what was done in this PR. (Examples Below)
- Fix HTML validation errors/warnings, check comments for more details.
  > Validate your HTML markup with https://validator.w3.org/#validate_by_upload.
- Nothing is messed up. See comments below on some messed up part of the page and suggested fixs.
- See other comments for required changes.

Kindly, check the comment below for suggestions.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Review by bolah2009 — COMMENTED** [review]
# Rename PR

  rename PR

-----

# Update PR description

  update-PR-description-html

File: src/index.html (hunk 1 above)
Comment (yirano) [reply src/index.html]: Resolved.

File: src/style.css (hunk 3 above)
Comment (yirano) [reply src/style.css]: Resolved.

File: src/index.html (hunk 2 above)
Comment (tiagoit) [reply src/index.html]: Resolved.

File: src/style.css (hunk 4 above)
Comment (tiagoit) [reply src/style.css]: Resolved.

**Review by whiz25 — APPROVED** [review]
## Awesome job :clap: team @yirano and @ferreirati 
You have met all requirements needed for the project.
I wish you success in your next projects.
Please use ```Afani Martin``` when filling the project completion form.

**Pull request merged**